src/installer/tests/HostActivation.Tests/PortableAppActivation.cs

```diff
@@ -554,6 +554,40 @@ public void AppHost_GUI_FrameworkDependent_MissingRuntimeFramework_ErrorReported
             }
         }
 
+        [Fact]
+        [PlatformSpecific(TestPlatforms.Windows)]
+        public void AppHost_GUI_RuntimeMissing_PrintArchitectureAndRuntimeVersionOnDialog()
```

Comment:
> PrintArchitectureAndRuntimeVersionOnDialog

The test isn't actually checking what is on the dialog, right? Just checking that a dialog is shown and that we log about showing it and the URL.

Reply:
Correct. AFAIK we currently don't have any tests checking the actual text on the dialog. ~Renaming.